ui/app/js/controllers/permission_scheduled_changes_controller.js

```diff
@@ -0,0 +1,179 @@
+angular.module("app").controller("PermissionScheduledChangesController",
+function($scope, $routeParams, $location, $timeout, Permissions, Rules, Search, $modal, $route, Releases) {
+
+  $scope.loading = true;
+  $scope.failed = false;
+
+
+
+  Permissions.getScheduledChanges()
+  .success(function(response) {
+    // "when" is a unix timestamp, but it's much easier to work with Date objects,
+    // so we convert it to that before rendering.
+    $scope.scheduled_changes = response.scheduled_changes.map(function(sc) {
+      if (sc.when !== null) {
+        sc.when = new Date(sc.when);
+      }
+      return sc;
+    });
+  })
+  .error(function() {
+    console.error(arguments);
+    $scope.failed = true;
+  })
+  .finally(function() {
+    $scope.loading = false;
+  });
+
+
+  $scope.$watch("ordering_str", function(value) {
+    $scope.ordering = value.value.split(",");
+  });
+
+
+   if ($scope.sc_id) {
+    $scope.ordering_options = [
+      {
+        text: "Data Version",
+        value: "-data_version"
+      },
+    ];
+  } else {
+    $scope.ordering_options = [
+
```

Comment:
Since there's only one ordering option, it's better to just leave this out and hide the dropdown.